src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -2108,7 +2108,7 @@ def get_variable_OM_costs(
         )
 
         # assume the user is not using this
-        b.other_variable_costs.fix(0)
+        b.other_variable_costs.fix(1e-12)
```

Comment:
Again, why fix this to such a small value?

Reply:
See above.